Frontend/Pages/WSE/WseDetail.razor

```diff
@@ -1,6 +1,6 @@
-﻿@page "/wse"
+﻿@page "/wse/{WseId:long}"
 @using Models
-@using Models.DTO
+@inject HttpClient HttpClient
```

Comment:
I personally think using `[Inject]` in the code block is superior as it's much closer to where it's actually used than here